shared-bindings/storage/__init__.c

```diff
@@ -48,16 +48,18 @@
 //| directly.
 //|
 
-//| .. function:: mount(filesystem, mount_path, \*, readonly=False)
+//| .. function:: mount(filesystem, mount_path, \*, readonly=False, disable_concurrent_write_protection=False)
```

Comment:
`disable_concurrent_write_protection` is listed as a new arg here for `mount()`, but it's not implemented. Did you mean to add it as with `remount()`, or is this a typo?

Reply:
Yup, a bug. I thought I had fixed that but probably forgot to save.